src/iquest.cpp

```diff
@@ -51,6 +51,11 @@ usage() {
         "in the 'WHERE' condition will be made upper case so one can do case-insensitive",
         "tests (using upper-case literals).",
         " ",
+        "To do a case-insensitive search, pass the 'uppercase' (or 'upper') option and",
+        "make ALL conditional input values uppercase.  For example:",
+        " ",
+        "  iquest uppercase \"SELECT DATA_ID WHERE COLL_NAME LIKE '%/TEST' AND DATA_NAME = 'FOO.BIN'\"",
```

Comment:
Not sure if it's worth noting a ~bug~ limitation, but if anything in the `WHERE` clause includes numbers (e.g. `DATA_SIZE`), GenQuery will return a `CAT_SQL_ERR` when used with `upper`/`uppercase`.

Reply:
Yes, will definitely include that.

Reply:
Sorry, I meant to say if any of the conditions are supposed to be interpreted as numbers. If a `DATA_NAME`, for instance, has a number, that's okay. I think you got it. :p

Reply:
Done. Please see new commit for new text.

Reply:
words seem good.

is it too much to make it NOT break in that way?   or that's for later...

Reply:
I discovered this issue when investigating fixing `upper`/`uppercase` for `DATA_RESC_HIER` in https://github.com/irods/irods/issues/6912. I guess we never had a formal conversation/decision-making process about it.

My vote is that this will be better in GenQuery2 and we leave it alone for GenQuery1.

Reply:
I agree that we leave GenQuery1 as-is. GenQuery2 is right around the corner and things like this give users a reason to reach for it and provide feedback.

Reply:
yeps - all good.   document and move on down the line.